cedar-drt/fuzz/fuzz_targets/level-validation-drt.rs

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Cedar Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#![no_main]
+use cedar_drt::*;
+use cedar_drt_inner::*;
+use cedar_policy_core::ast;
+use cedar_policy_generators::{abac::ABACPolicy, schema::Schema, settings::ABACSettings};
+use libfuzzer_sys::arbitrary::{self, Arbitrary, Unstructured};
+use log::{debug, info};
+use serde::Serialize;
+
+/// Input expected by this fuzz target
+#[derive(Debug, Clone, Serialize)]
+pub struct FuzzTargetInput {
+    /// generated schema
+    #[serde(skip)]
+    pub schema: Schema,
```

Comment:
We removed `Serialize` from other `FuzzTargetInput`s, and in https://github.com/cedar-policy/cedar-spec/pull/572 I think we decided not to add it back.  Probably we should leave `Serialize` off for this new target?